appintro/src/main/java/com/github/paolorotolo/appintro/internal/PermissionWrapper.kt

```diff
@@ -10,7 +12,7 @@ internal data class PermissionWrapper(
     var permissions: Array<String>,
     var position: Int,
     var required: Boolean = true
-) {
+) : Serializable {
```

Comment:
Just one question, is this Serializable necessary, i couldn't find its use anywhere...

Reply:
It's necessary as we're parcelizing an `HashMap<Int, PermissionWrapper>`. The `HashMap` is actually serializable so you can use `.putSerializable`, but the `PermissionWrapper` is not. It crashes at runtime during the `onResume()` when trying to deserialise such class.